packages/content-conformance/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "moduleResolution": "NodeNext",
     "jsx": "react",
     "outDir": "./dist",
-    "declaration": true
+    "declaration": true,
+    "esModuleInterop": true,
```

Comment:
These trailing commas help with reducing noise in PRs, and providing proper blame in git.